src/Core/Domain/Cart/QueryResult/CartForOrderCreation/CartProduct.php

```diff
@@ -206,6 +222,22 @@ public function getCustomization(): ?Customization
         return $this->customization;
     }
 
+    /**
+     * @return int
+     */
+    public function getStock(): int
```

Comment:
This should be `getStockAvailable` for consistency

Reply:
What does this return when stock management is disabled?

Reply:
Renamed for the variable name.

Reply:
For the second, I think it's a complete regression.

Reply:
And it doesn't seem a regression
So it's a bug : https://github.com/PrestaShop/PrestaShop/issues/23216